server/src/main/java/org/spine3/server/reflect/FailureSubscriberMethod.java

```diff
@@ -145,8 +145,8 @@ static FailureSubscriberMethod from(Method method) {
         return new FailureSubscriberMethod(method);
     }
 
-    private static IllegalStateException missingFailureHandler(Class<?> cls,
-                                                               Class<? extends Message> failureClass) {
+    private static IllegalStateException
+    missingFailureHandler(Class<?> cls, Class<? extends Message> failureClass) {
```

Comment:
I'm not sure too. 
@alexander-yevsyukov Can you please tell you opinion about what will be the better way to wrap such cases?